xformers/components/attention/csrc/cuda/mem_eff_attention/kernels/generate_kernels.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
It would totally make sense if we start generating more code, but as the number of files is quite limited, I believe it helps readability (and is simpler) to have the files already generated in the repo. But totally up to you